README.md

```diff
@@ -137,6 +137,22 @@ const { total } = await myDocumentDao.decr(
 );
 ```
 
+When multiple values must be incremented and/or decremented in the same call:
+
+```ts
+// `total` will have the value `5` and `extra` will have the value -1.
```

Comment:
`total` and `extra` as in the record, or `total` and `extra` as in the variables declared here?